src/ecosystem/corals/GrowthAcceleration.jl

```
@@ -0,0 +1,116 @@
+function growth_acceleration_values(param_type::Symbol)
+    if param_type == :steepness
+        return growth_acceleration_steepness()
+    elseif param_type == :height
+        return growth_acceleration_height()
+    elseif param_type == :midpoint
+        return growth_acceleration_midpoint()
+    end
+    return nothing
```

Is it better to throw an exception?